packages/story-utils/src/__tests__/getStories.test.ts

```diff
@@ -0,0 +1,50 @@
+import getStories from '../getStories';
+
+test('finding stories', () => {
+  const stories = getStories('packages/story-utils/**/*.stories.*');
+
+  expect(stories).toEqual([
+    {
+      componentTitle: 'Bar',
+      name: 'BarStory1',
+      parameters: expect.any(Object),
+      storyFn: expect.any(Function),
+    },
+    {
+      componentTitle: 'Foo',
+      name: 'FooStory1',
+      parameters: expect.any(Object),
+      storyFn: expect.any(Function),
+    },
+    {
+      componentTitle: 'Foo',
+      name: 'FooStory2',
+      parameters: expect.any(Object),
+      storyFn: expect.any(Function),
+    },
+  ]);
+});
+
+test('merging parameters', () => {
+  const stories = getStories('packages/story-utils/**/foo.stories.*');
+
+  expect(stories).toEqual([
+    expect.objectContaining({
+      name: 'FooStory1',
+      parameters: {
+        axe: {
+          disabled: true,
+          rules: ['color-contrast'],
+        },
+      },
```

Comment:
Wish we could use `mock-fs` to define the files in memory in these tests, so that it's clear where these parameters are coming from.

But mock-fs doesn't seem to work right with dynamic require statements (particularly on Linux and Windows), and I opted to use real files for these test stories.